lib/options.py

```diff
@@ -511,8 +511,36 @@ def __init__(self):
 
         self.check()
 
+        self.correct_timeout_offset(parser)
+
         Options._initialized = True
 
+    def correct_timeout_offset(self, parser: argparse.ArgumentParser) -> None:
+        """Check and correct timeout offset if necessary."""
+        default_time_offset = 10
+        if self.args.server_start_timeout > parser.get_default('server_start_timeout'):
+            if (self.args.test_timeout - self.args.server_start_timeout) < default_time_offset:
+                color_stdout("Warning: test-timeout and no-output-timeout has been changed. \n",
```

Comment:
I think this message does **not** answer the question "why".
The values changed automatically, why? What should the user do to get rid of this warning?

Reply:
Fixed